.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,84 @@
+name: Feature Request
+description: Suggest an idea or a general improvement.
+labels: ["kind/feature"]
+body:
+  - type: dropdown
+    id: Component
+    attributes:
+      label: Component
+      description: For which component your improvement idea is for?
+      options:
+        - Not sure-
+        - Operate-
+        - Optimize-
+        - Tasklist-
+        - Identity-
+        - Zeebe-
+        - C8Run-
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: user-story
+    attributes:
+      label: User Story
+      description: Please describe the problem you are trying to solve
+    validations:
+      required: true
+  - type: textarea
+    id: design
+    attributes:
+      label: Design
+      description: Please describe the solution you would like to see implemented
+    validations:
+      required: true
+  - type: textarea
+    id: functional-requirements
+    attributes:
+      label: Functional requirements
+    validations:
+      required: false
+  - type: textarea
+    id: technical-requirements
+    attributes:
+      label: Technical requirements
+    validations:
+      required: false
+  - type: textarea
+    id: dev-qa-handover
+    attributes:
+      label: Dev -> QA handover
+      description:
+      value: |
+             - Resources:
+             - Versions to validate:
+             - Release version (in which version this feature will be released):
+  - type: textarea
+    id: qa-test
+    attributes:
+      label: QA test case
+    validations:
+      required: false
+  - type: textarea
+    id: links
+    attributes:
+      label: Links
+      description: Add the support ticket- or other links here if applicable.
+    validations:
+      required: false
+  - type: textarea
+    id: breakdown
```

Comment:
💭 won't this be done with the "new" subissues feature?

Reply:
Yes, I think we can remove those sections. I made the suggestion when subtasks were just starting as a new feature and didn't factor that in. A dedicated section in the template might encourage more active thinking about breaking down, but I think that's not a strong enough reason to keep it.